test-src/api_test.js

```diff
@@ -0,0 +1,174 @@
+let workerType = require('../lib/worker-type');
```

Comment:
We already have a file that tests the api, it's called manageworkertype_test.js.  I don't think there's any reason for these tests to be in a different file.